docker-compose.yml

```diff
@@ -11,7 +11,9 @@ services:
       DJANGO_SETTINGS_MODULE: "pycon.dev_settings"
       SECRET_KEY: "secret"
     command: ["./manage.py migrate --no-input && ./manage.py runserver 0.0.0.0:8888"]
+    # command: ["python -m ptvsd --host 0.0.0.0 --port 3000 --wait ./manage.py runserver --noreload --nothreading 0.0.0.0:8888"]
```

Comment:
I would recommend having these 2 commands in a single `start.sh` shell file and they would get executed depending on a environment variable you set at runtime (or in the `.env` file). Then you could do something like:
```bash
$ VSCODE_DEBUG=1 docker-compose up
```
and it would enter in debug mode.

Reply:
Thanks for the suggestion with the env variable, that's much better than manually commenting the code! Stéphane suggested to use docker override and I think that was even better idea (feels more in line with Docker), so I've changed that.